<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinanceManagerView</class>
 <widget class="QMainWindow" name="FinanceManagerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сoncierge: Finance</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabLog">
        <attribute name="title">
         <string>Log</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>471</width>
           <height>331</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vLayStatistic">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>70</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>New Row</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>New Row</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>New Row</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Date</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>19.06.13</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>Перевод с &lt;карта1&gt; на &lt;карта2&gt; 500 р. (скинул в &lt;банк2&gt;)</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>20.06.13</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>Заработал 2000 р. (ЗП &lt;работа1&gt;)</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>21.06.13</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>Потратил 500 р. (кофейня)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabMian">
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="gbAccounts">
            <property name="title">
             <string>Счета</string>
            </property>
            <widget class="QWidget" name="formLayoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>181</width>
               <height>268</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="fLayAccounts">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Наличка</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QProgressBar" name="progressBar">
                <property name="value">
                 <number>24</number>
                </property>
                <property name="textVisible">
                 <bool>true</bool>
                </property>
                <property name="format">
                 <string notr="true">%v р.</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Карта 1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QProgressBar" name="progressBar_2">
                <property name="value">
                 <number>24</number>
                </property>
                <property name="format">
                 <string notr="true">%v р.</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Карта 2</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QProgressBar" name="progressBar_3">
                <property name="value">
                 <number>24</number>
                </property>
                <property name="format">
                 <string notr="true">%v р.</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Карта 3</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QProgressBar" name="progressBar_4">
                <property name="value">
                 <number>8</number>
                </property>
                <property name="format">
                 <string notr="true">%v р.</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Карта 4</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QProgressBar" name="progressBar_5">
                <property name="value">
                 <number>24</number>
                </property>
                <property name="format">
                 <string notr="true">%v р.</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Карта 5</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QProgressBar" name="progressBar_6">
                <property name="value">
                 <number>24</number>
                </property>
                <property name="format">
                 <string notr="true">%v р.</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Вклад 1</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QProgressBar" name="progressBar_7">
                <property name="value">
                 <number>50</number>
                </property>
                <property name="format">
                 <string notr="true">%v р.</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Вклад 2</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QProgressBar" name="progressBar_8">
                <property name="value">
                 <number>70</number>
                </property>
                <property name="format">
                 <string notr="true">%v р.</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Итого :</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QSpinBox" name="spinBox">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="suffix">
                 <string> р.</string>
                </property>
                <property name="minimum">
                 <number>-100000</number>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="value">
                 <number>500</number>
                </property>
               </widget>
              </item>
              <item row="8" column="0" colspan="2">
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QDateEdit" name="deCurrentDate">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="dateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2013</year>
                <month>5</month>
                <day>30</day>
               </datetime>
              </property>
              <property name="displayFormat">
               <string>ddd, dd MMMM yyyy года</string>
              </property>
              <property name="date">
               <date>
                <year>2013</year>
                <month>5</month>
                <day>30</day>
               </date>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbNewTransaction">
              <property name="text">
               <string>Новая транзакция</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Сегодня</string>
              </property>
              <widget class="QWidget" name="formLayoutWidget_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>20</y>
                 <width>209</width>
                 <height>141</height>
                </rect>
               </property>
               <layout class="QFormLayout" name="fLayToday">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Проезд :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox">
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="accelerated">
                   <bool>false</bool>
                  </property>
                  <property name="correctionMode">
                   <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                  </property>
                  <property name="keyboardTracking">
                   <bool>true</bool>
                  </property>
                  <property name="suffix">
                   <string> р.</string>
                  </property>
                  <property name="minimum">
                   <double>-100000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>100000.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>-20.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>ЗП:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="accelerated">
                   <bool>false</bool>
                  </property>
                  <property name="correctionMode">
                   <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                  </property>
                  <property name="keyboardTracking">
                   <bool>true</bool>
                  </property>
                  <property name="suffix">
                   <string> р.</string>
                  </property>
                  <property name="minimum">
                   <double>-100000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>100000.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>-20.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionHistory"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
